source/simulator/parameters.cc

```diff
@@ -913,6 +913,16 @@ namespace aspect
                          "this way, then it does not evolve at all. Its values are "
                          "simply set to the initial conditions, and will then "
                          "never change."
+                         "\n"
+                         "\\item ``melt field'': If a compositional field is marked with this "
```

Comment:
it also includes diffusion, right?

Reply:
No, I don't think so. For compositional fields the diffusivity is always zero.

Reply:
oh. :-)